whelk-core/src/main/groovy/whelk/converter/marc/MarcFrameConverter.groovy

```diff
@@ -1402,18 +1403,21 @@ class MarcFixedFieldHandler {
         int end
         Integer itemPos
         String fixedDefault
+        Boolean ignoreOnRevert = null
         Pattern matchAsDefault
         MarcFixedFieldHandler fixedFieldHandler
 
         Column(MarcFixedFieldHandler fixedFieldHandler, fieldDfn, int start, int end,
-               itemPos, fixedDefault, matchAsDefault = null) {
+               itemPos, fixedDefault, ignoreOnRevert, matchAsDefault = null) {
```

Comment:
Perhaps initialize to null in the same way as 'matchAsDefault' to not mix conventions? If it needs to be initialized to null at all...